CHANGELOG.md

```diff
@@ -1,10 +1,86 @@
 # Changelog
 
+## 7.0.0
```

Comment:
@lukechesser I added a changelog for v7 and factored in all your feedback. Since the breaking changes are fairly substantial, I often pointed to the updated README.

Let me know if you see anything in particular that's missing.

Reply:
Perfect — this looks great to me 👍  Straightforward